modules/dhcp_common/isc/omapi_provider.rb

```diff
@@ -6,11 +6,12 @@ class IscOmapiProvider < ::Proxy::DHCP::Server
     include Proxy::Util
     attr_reader :omapi_port, :key_name, :key_secret
 
-    def initialize(server, omapi_port, subnets = nil, key_name = nil, key_secret = nil, service = nil, free_ips_service = nil)
+    def initialize(server, omapi_port, subnets = nil, key_name = nil, key_secret = nil, service = nil, free_ips_service = nil, key_algorithm = nil)
```

Comment:
We should really refactor this to keyword arguments, but I wanted to stay compatible.